.clang-format

```diff
@@ -0,0 +1,8 @@
+BasedOnStyle: LLVM
+IndentWidth: 4
```

Comment:
FWIW, most comments are formated so the line length stays under 80 columns ... when indent is 3

Reply:
those are the ones with tab indents ... when space indents are used, many files use 4 spaces...

Reply:
Nowadays people argue that it is better to wrap at something larger than 80 because people have larger, and particularly wider screens, so it makes sense to relax this constraint somewhat. I do not want to argue for this though, I can change it to your liking.

Reply:
I don't really care, my editor ignored clang-format ... my screens are larger, but that doesn't mean I have more columns ... I just have LOTS more windows ... all 80 columns wide.  That way, I don't have to flip around, everything is there in front of me.